_data/transport_integrations.yml

```diff
@@ -0,0 +1,51 @@
+- type: Collector
+  library: >-
+    [zipkin-aws collector-sqs](https://github.com/openzipkin/zipkin-aws/tree/master/autoconfigure/collector-sqs)
+  product: >-
+    [AWS SQS](https://aws.amazon.com/sqs/)
+  notes:
+
+- type: Collector
+  library: >-
+    [zipkin-aws collector-kinesis](https://github.com/openzipkin/zipkin-aws/tree/master/autoconfigure/collector-kinesis)
+  product: >-
+    [AWS Kinesis](https://aws.amazon.com/kinesis/)
+  notes:
+
+- type: Storage
+  library: >-
+    [zipkin-aws storage-elasticsearch-aws](https://github.com/openzipkin/zipkin-aws/tree/master/autoconfigure/storage-elasticsearch-aws)
+  product: >-
+    [AWS Elasticsearch Service](https://aws.amazon.com/elasticsearch-service/)
+  notes:
+
+- type: Storage
+  library: >-
+    [zipkin-aws storage-xray](https://github.com/openzipkin/zipkin-aws/tree/master/autoconfigure/storage-xray)
+  product: >-
+    [AWS X-Ray](https://aws.amazon.com/ko/xray/)
+  notes: only supports sending to an XRay UDP daemon
+
+- type: Collector
```

Comment:
remove this because it is no longer released

Reply:
removed.